vendor/github.com/docker/docker/api/types/types.go

```diff
@@ -327,7 +328,7 @@ type ContainerJSONBase struct {
 	Name            string
 	RestartCount    int
 	Driver          string
-	Platform        string
+	Platform        specs.Platform
```

Comment:
This change will be reverted again after https://github.com/moby/moby/pull/35144 is merged